modules/rostests/apitests/acpi/CMakeLists.txt

```diff
@@ -1,9 +1,13 @@
 
 include_directories(${REACTOS_SOURCE_DIR}/drivers/bus/acpi/acpica/include)
 
+generate_whitelist_code(acpi_apitest_whitelist.txt)
+add_definitions(-DUSE_WHITELIST)
```

Comment:
Do you have a recommendation for how we should handle tests like this that aren't expected to succeed on Windows?
This is a real unit test, so it always runs our code, even on Windows.

We could move it to a new unittests subdirectory to make this clearer. And/or maybe exclude it from the list of tests we run on WHS?
Or do you think just having whitelist entries is a good permanent solution?

Reply:
I suggest to skip those tests on Windows that are known to be broken. Or fix the code in ros, so that they succeed ;-)
The whitelists are generally not for our own tests, but for wine-tests, so we can silence them without hacking the code.
In our own tests I suggest using `todo`.